net/procfs/net_procfs.c

```diff
@@ -244,7 +246,7 @@ static int netprocfs_open(FAR struct file *filep, FAR const char *relpath,
 
       devname = basename(copy);
       dev     = netdev_findbyname(devname);
-      kmm_free(copy);
+      lib_free(copy);
```

Comment:
but, procfs is always bundled in the kernel image, so lib_malloc in strdup is always mapped to kmm_malloc.

Reply:
@royfengsss see this patch: https://github.com/apache/nuttx/pull/10066